website/docs/d/cognitive_account.html.markdown

```diff
@@ -35,39 +35,113 @@ The following arguments are supported:
 
 The following attributes are exported:
 
+* `custom_question_answering_search_service_id` - The ID of the search service. This is only available when the `kind` is `TextAnalytics`.
```

Comment:
`id` is a Common Azure field and is always exported, so it should always be the first field in the attributes section. Data source documentation should never explain the field, that should be done in the resource documentation, here we should just explain what the field is. So, we need to update `custom_question_answering_search_service_id` description to remove the explanation.
```suggestion
* `id` - The ID of the Cognitive Services Account. 

* `custom_question_answering_search_service_id` - The ID of the search service.
```

Reply:
Updated